composer.json

```diff
@@ -53,6 +53,12 @@
   "extra": {
     "tao-extension-name": "generis"
   },
+  "repositories": [
```

Comment:
repositories added in non root composer.json will be ignored an lead to confussion